log-sink/src/main/java/org/springframework/cloud/stream/module/log/LogSink.java

```diff
@@ -16,25 +16,33 @@
 
 package org.springframework.cloud.stream.module.log;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.cloud.stream.annotation.EnableBinding;
 import org.springframework.cloud.stream.messaging.Sink;
+import org.springframework.context.annotation.Bean;
 import org.springframework.integration.annotation.ServiceActivator;
+import org.springframework.integration.handler.LoggingHandler;
 
 /**
  * @author Dave Syer
  * @author Marius Bogoevici
+ * @author Gary Russell
  */
 @EnableBinding(Sink.class)
+@EnableConfigurationProperties(LogSinkProperties.class)
 public class LogSink {
 
-	private static Logger logger = LoggerFactory.getLogger(LogSink.class);
+	@Autowired
+	private LogSinkProperties config;
```

Comment:
we're still not 100% consistent across modules, but in most cases I think we're referring to the properties instance as `properties`